snmp/datadog_checks/snmp/data/conf.yaml.example

```
@@ -1,99 +1,112 @@
+    ## The number of OIDs handled by each batch. Increasing this number improves performance but
+    ## uses more resources.
+    ##
+    ## In corecheck, the default value depends on the version of the agent.
+    ## Before version 7.30 the default value is 60 and from version 7.30 its value is 5.
+    #
+    # oid_batch_size: 10
+
+    ## @param global_metrics - list of mappings - optional
+    ## Specify global_metrics you want to monitor by using MIBS for counter and gauge.
+    ## global_metrics are applied to all instances where use_global_metrics is set to true at the instance level.
+    ##
+    ## Single scalar metrics and table metrics can be defined in this section.
+    ## Details on how to define metrics can be found in the documentation:
+    ## https://datadoghq.dev/integrations-core/tutorials/snmp/profile-format/#metrics
+    #
+    # global_metrics:
+    #   - OID: 1.3.6.1.2.1.6.5.0
+    #     name: tcpPassiveOpens
+    #   - OID: 1.3.6.1.2.1.6.5.0
+    #     name: tcpPassiveOpens
+    #     metric_tags:
+    #     - TCP
+
+    ## @param profiles - mapping - optional
+    ## Specify profiles to be able to reference a set of metrics by name.
+    ## Either `definition_file` or `definition` must be defined.
+    ## `definition_file` points to a file with a profile, and `definition` inlines
+    ## it directly.
+    ## When using a profile present on the profiles directory of the
+    ## configuration, you can directly pass the filename.
+    ## By default, if the file doesn't start with an underscore, it loads all of the profiles
+    ## installed in the python package and in the configuration directory.
+    #
+    # profiles:
+    #   f5-big-ip:
+    #     definition_file: f5-big-ip.yaml
+    #   router:
+    #     definition_file: generic-router.yaml
 
+    ## @param refresh_oids_cache_interval - integer - optional - default: 0
+    ## Note: Beta feature, only available using python SNMP integration.
+    ## Set this option to enable caching of OIDs. The value is the number of seconds before the
+    ## the cache is refreshed.
+    ## This option works best when `oid_batch_size` is set to 128 or 256.
+    ## This is useful to improve performance when fetching metrics from large tables.
+    ## When OID caching is enabled, table row OIDs are reused. This means that the table can be fetched
+    ## using batched GET queries, as opposed to one GETNEXT query per row in the table.
+    ## The caching is disabled if the interval is set to `0` (default).
+    ## Note: If some tables rows are dynamic, new rows are only fetched every refresh interval.
+    ##         - This means new OIDs are also only picked up every refresh interval.
+    ##         - With auto discovery, if the same IP is assigned to a different device with different OIDs,
+    ##         the OIDs are refreshed only according to the refresh interval.
+    ## Only available using python SNMP integration
+    #
+    # refresh_oids_cache_interval: 0
```

```suggestion
    # refresh_oids_cache_interval: 3600
```

Let's keep the same display value as before